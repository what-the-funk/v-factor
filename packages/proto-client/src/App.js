import React from 'react';
import { Route } from 'react-router-dom';
// import { SecureRoute, ImplicitCallback } from "@okta/okta-react";

import Navigation from './navigation/components/Navigation';
import HomePage from './home/components/HomePage';
// import LoginPage from "./login/components/LoginPage";
// import RegistrationForm from "./registration/components/RegistrationForm";
// import ProfilePage from "./profile/components/ProfilePage";
import StreamerPage from './stream/components/StreamerPage';
import ViewerPage from './stream/components/ViewerPage';

// const { REACT_APP_OKTA_ORG_URL } = process.env;

const App = () => (
  <div>
    <Navigation />
    <main>
      <Route path="/" exact component={HomePage} />
      {/* <Route path="/implicit/callback" component={ImplicitCallback} />
      <Route path="/login" render={() => <LoginPage baseUrl={REACT_APP_OKTA_ORG_URL} />} />
      <Route path="/register" component={RegistrationForm} />
      <SecureRoute path="/profile" component={ProfilePage} /> */}
      <Route path="/stream/:room?" component={StreamerPage} />
      <Route path="/view" component={ViewerPage} />
    </main>
  </div>
);

export default App;
