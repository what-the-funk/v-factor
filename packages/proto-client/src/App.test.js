import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";

import App from "./App";

it("snapshot test", () => {
  let wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
