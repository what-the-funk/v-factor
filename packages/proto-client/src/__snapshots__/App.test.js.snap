// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div>
  <withRouter(Navigation) />
  <main>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/stream/:room?"
    />
    <Route
      component={[Function]}
      path="/view"
    />
  </main>
</div>
`;
